### ISSUE TITLE
Transfer meetup.com

### ISSUE THREAD

**Issue by mtaylorlong** [maptimemsp/organizers#1]
I've been lording over the meetup.com group. I can step down as an organizer, but I need to transfer ownership to one of you so someone else can handle fees in the future.

**Comment by mtaylorlong** [comment]
I just stepped down as the "Organizer" of our meetup.com group and suggested a transfer to Liz that she will have to confirm.

@lpuhl: The payment process is not connected to a group, rather an individual organizer. To be an organizer of a large group, you'll have to pay for the unlimited plan. I suggest setting up the payment ASAP to keep the group afloat, then soliciting sponsorships/donations to pay yourself back.

**Comment by mtaylorlong** [comment]
I also added @geospatialem and @bitner as Co-organizers of the meetup.com group. As co-organizers, you can do everything Liz can except for accepting payments. If Liz wants to get compensation for her meetup fees, she can set up her account to take money from the donations button and can prompt sponsors to use that.

**Comment by fawcett** [comment]
If Liz accepts the official Organizer crown, I am willing to guarantee at least 3 months worth of payment if we fall short on sponsorship funds.  I want to ensure that she doesn't have to worry about getting stuck with a lot of fees.

**Comment by mtaylorlong** [comment]
I just checked back in on the meetup page and it looks like I'm still the organizer, which means @lpuhl didn't accept in time. Perhaps Liz isn't getting notifications from meetup or for github (or perhaps she doesn't want to be stuck with the meetup.com page?). In any case, I made another attempt to transfer it to her tonight. If she doesn't accept, would any of the rest of you be willing to own the meetup.com group?
